concepts/felts/about.md

```diff
@@ -1 +1,26 @@
 # Felt Type
```

Comment:
Please update the _.meta/config.json_ file in the following way:
- remove the existing author (`misicnenad`) and add your GH handle
- add the missing [blurb](https://exercism.org/docs/building/tracks/concepts#h-documentation-files)